packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -121,6 +132,25 @@ export interface ServerDeploymentGroupProps {
      * @default ServerDeploymentConfig#OneAtATime
      */
     deploymentConfig?: IServerDeploymentConfig;
+
+    /**
+     * The auto-scaling groups belonging to this Deployment Group.
+     *
+     * @default []
+     */
+    autoScalingGroups?: autoscaling.AutoScalingGroup[];
```

Comment:
Not sure we have it, but can we use `AutoScalingGroupRef`?

Reply:
We do not.

Reply:
#752